### ISSUE TITLE
Design Alignment: Review all Cal-ITP basic components for opportunities to re-use

### ISSUE THREAD

**Issue by srhhnry** [cal-itp/benefits#1466]

**Comment by machikoyasuda** [comment]
It might be worth going through Benefits app and Admin app, and noting what components/styles (type, buttons, header, radio buttons, etc.) are worth aligning between the two. @ohmegasquared @indexing

**Comment by ohmegasquared** [comment]
Yes! I have started this process for a holistic design system with calitp.org, too. It takes some time to work through it all! I hope to pick it back up after I finish the benefit admin work. It may require some refactoring to get them aligned after the first pass.
